index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Resume</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <header>
+        <h1>Resume Siarhei Matsiash</h1>
+        <img src="foto.jpg" alt="foto" class="foto">
+    </header>
+    <main>
+        <article>
+            <h4>1. Name and surname:</h4>
```

Comment:
If you want to create a list with content it is always a good idea to use `<ol>/<ul>`.